traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java

```diff
@@ -523,7 +530,11 @@ public Geolocation getLocation(final String clientIP, final DeliveryService deli
 			track.setResultDetails(ResultDetails.GEO_NO_CACHE_FOUND);
 		}
 
-		track.setResult(ResultType.GEO);
+		if (useDSDefaults) {
```

Comment:
would this be able to go into the if statement above? instead of having the useDSDefaults boolean just set the result type in that if statement instead? then we could get rid of the SuppressWarnings too

Reply:
Done 👍